terraform/modules/aws_gundeck-push-notifications/README.md

```diff
@@ -0,0 +1,23 @@
+Terraform module: Gundeck Push Notifications
+============================================
+
+This module enables Push Notifications for iOS and Android devices. It is used by Gundeck when running in an AWS cloud.
+
+
+#### How to use the module
+
+```hcl-terraform
+module "gundeck-push-notification" {
+  source = "github.com/wireapp/wire-server-deploy.git//terraform/modules/aws_gundeck-push-notifications?ref=develop"
+  
+  region = "eu-central-1"
+  environment = "dev"
+  account_id = ""
+  apns_application_id = "wire.com"
+  apns_credentials_path = "${path.root}/path/to/app-credentials/files"   # omit '.[cert,key].pem'
```

Comment:
Can we assume that the files will be named `voip`? This shows that it's a `voip` certificate (there are other types but we are not using them currently
```suggestion
  apns_credentials_path = "${path.root}/path/to/app-credentials/files"   # omit 'voip.[cert,key].pem'
```

Reply:
Hard-coding this, I think, has no benefit, but also doesnt prevent you form naming the files that way. I'm even hesitant to use the path to the files as variable. I'd would rather have the file content itself to be the variable value. I only decided otherwise, because then we would need another var for APNS, since it requires to files for credentials.
But the more I'm thinking about it, the more I'm open to this, because it's more explicit and visible.

Reply:
I think hardcoding and forcing some convention is fine; if not, then we should at least make it explicit what sort of credentials we are looking for.

Reply:
I decided to go with providing file content instead of the path and ended up with 3 variables overall (2 apns, 1 gcm). the readme shows how to use it. Therefore, the modules does not impose any file naming. It is up to the user how to organize their files.